notebooks/Getting_started_with_Forecast.ipynb

```diff
@@ -390,31 +384,21 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "forecast.describe_dataset_import_job(DatasetName=datasetName,VersionId=ds_versionId)"
+    "forecast.describe_dataset_import_job(DatasetImportJobArn=ds_import_job_arn)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Recipe"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "recipesResponse=forecast.list_recipes()\n",
-    "recipesResponse"
+    "### Create Solution with customer forecast horizon"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Get details about each recipe."
+    "Forecast horizon is how long in future the forecast should be predicting. For weekly data, a value of 12 means 12 weeks. Our example is hourly data, we try forecast the next day, so we can set to 24."
```

Comment:
nit: Over use of word forecast. Forecast horizon is the number of number of time points to predicted in the future.

Reply:
Done.